packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -449,7 +452,8 @@ export async function runNpmInstall(
     debug(`Installing to ${destPath}`);
 
     const opts: SpawnOptionsExtended = { cwd: destPath, ...spawnOpts };
-    const env = opts.env ? { ...opts.env } : { ...process.env };
+    const envObj = opts.env || process.env;
+    const env: typeof process.env = { PATH: envObj.PATH, ...envObj };
```

Comment:
**question (soft-blocking):** Do we also need to change these places?

```
packages/cli/src/commands/bisect/index.ts:
  288:           ...process.env,

packages/cli/src/util/dev/builder.ts:
  68:     ...process.env,

packages/cli/test/dev/utils.js:
  391:         ? { ...process.env, __VERCEL_SKIP_DEV_CMD: 1 }

packages/go/go-helpers.ts:
  126:     ...process.env,

packages/go/index.ts:
  695:     ...process.env,

packages/ruby/index.ts:
  67:         ...process.env,

utils/run.js:
  94:           ...process.env,
```

Reply:
`packages/cli/src/commands/bisect/index.ts:`
Bisect is using `...process.env` when spawning `--run <cmd>` and assumes the cmd is resolvable.

`packages/cli/src/util/dev/builder.ts:`
On the next line after the `...process.env`, an overwritten `PATH` is being explicitly set.

`packages/cli/test/dev/utils.js:`
I didn't fix `...process.env` in tests. I assumed if the tests pass as is on my Windows machine, then no need to touch them.

`packages/go/go-helpers.ts:`
On the next line after the `...process.env`, an overwritten `PATH` is being explicitly set.

`packages/go/index.ts:`
I will fix this. I missed this one as the go builder doesn't have unit tests.

`packages/ruby/index.ts:`
I will fix this. I missed this one as the go builder doesn't have unit tests.

`utils/run.js:`
I'm not sure what calls this script. I'm not sure exactly how to test it. Do we call this script from any Windows machines? I'm leaning towards if it ain't broke, then don't fix it.